test/test_torchinductor_opinfo.py

```diff
@@ -91,7 +91,7 @@ def process(device_type):
     "mvlgamma.mvlgamma_p_5": {f32, f64, i32, i64},  # flaky
     "cumprod": {f32, f64},  # flaky
     "_masked.prod": {f32, f64},  # flaky
-    "empty_like": {b8, f16, f32, f64},  # flaky
+    "empty_like": {b8, f16, f32, f64, i32, i64},  # flaky
```

Comment:
lol we shouldn't be checking results of empty_like, they are undefined, but that's for another PR

Reply:
This test is flakey on master.